### ISSUE TITLE
Wake On Lan won't go through

### ISSUE THREAD

**Issue by northys** [safing/portmaster#564]
**What worked?**

When I turn off portmaster I can wake on lan my home server.

**What did not work?**

`wol` or `wakeonlan` command seems not working with portmaster turned on. I can't even see that binary in app list to check whether it was blocked somehow.

EDIT: just wanna clarify: the computer running portmaster is the one which sends the magic packet not the one being woken up

I don't understand WOL much (tried it first time today) but I suppose it somehow broadcasts some packets (or packet respectively) so I worry why it doesn't work while DLNA works. DLNA also use broadcast to get list of available servers.

**Comment by dhaavi** [comment]
I just read about the details of WoL for the first time. Wow. This is ugly. There are _so_ many ways of doing this, and it seems that some are very, very ugly. Bottom line: It could be _anything_.

I tried it out.

Ran this command (deb based):
```
$ wakeonlan 00:30:18:01:73:fb
Sending magic packet to 255.255.255.255:9 with 00:30:18:01:73:fb
```
(Not waking something real, address is from a tutorial.)

The Portmaster then saw this:
![image](https://user-images.githubusercontent.com/18184155/160150301-526f6527-ddc4-4aa5-8fb4-eb9bf0c53524.png)

**Comment by northys** [comment]
Yeah I can see the same under unidentified app. Do you have any idea why it is unidentified? I have the same problem with ping which I just wanted to exclude from SPN traffic because I need it all the time since I'm devops guy.

It works now actually... Maybe I was connected to SPN?

![Screenshot from 2022-03-26 21-33-06](https://user-images.githubusercontent.com/2878126/160256163-d55ee9f7-f8c7-4af0-9a4d-769c404242ca.png)

**Comment by dhaavi** [comment]
Please don't close issue if you want a reply. It will fall out of our filters and we won't see it anymore. ;)

> Do you have any idea why it is unidentified?

I guess by the time the Portmaster looks for the process, it has already vanished. It's a UDP fire and forget thing.

> I have the same problem with ping which I just wanted to exclude from SPN traffic because I need it all the time since I'm devops guy.

Just add a `Exclude * ICMP` rule to the SPN Rules of "Unknown App".

> It works now actually... Maybe I was connected to SPN?

Even so, LAN connections should always work, as they always bypass the SPN.

**Comment by northys** [comment]
> Just add a `Exclude * ICMP` rule to the SPN Rules of "Unknown App".

Thanks. I've added it globally just in case portmaster identifies ping in the future.

> Please don't close issue if you want a reply. It will fall out of our filters and we won't see it anymore. ;)

I just tried to be helpful, sorry. I also had similar question in SPN repo so I though we can continue there because WOL works.

**Comment by northys** [comment]
This works so I'm closing it. Thanks for further clarification.
